Flow.Launcher.Infrastructure/StringMatcher.cs

```diff
@@ -225,6 +213,22 @@ public MatchResult FuzzyMatch(string query, string stringToCompare, MatchOption
             return new MatchResult(false, UserSettingSearchPrecision);
         }
 
+        private bool IsAcronym(string stringToCompare, int compareStringIndex)
+        {
+            if (char.IsUpper(stringToCompare[compareStringIndex]) ||
+                    char.IsNumber(stringToCompare[compareStringIndex]) ||
+                    char.IsDigit(stringToCompare[compareStringIndex]))
+                return true;
+
+            if (compareStringIndex == 0)
+                return true;
+
+            if (compareStringIndex != 0 && char.IsWhiteSpace(stringToCompare[compareStringIndex - 1]))
```

Comment:
Please remove the duplicate check for 0 index.

Reply:
The first check is for when index is actually 0, and the other is a check to ensure that compareStringIndex-1 is not going to end up with index out of range exception